src/Xamarin.Android.Build.Tasks/Tasks/ProcessNativeLibraries.cs

```diff
@@ -22,6 +22,16 @@ public class ProcessNativeLibraries : AndroidTask
 			"libxamarin-debug-app-helper",
 		};
 
+		// Please keep the list sorted.  Any new runtime libraries that are added upstream need to be mentioned here.
+		static readonly HashSet<string> KnownRuntimeNativeLibrariesCLR = new (StringComparer.OrdinalIgnoreCase) {
+			"libSystem.Globalization.Native.so",
+			"libSystem.IO.Compression.Native.so",
+			"libSystem.Native.so",
+			"libSystem.Security.Cryptography.Native.Android.so",
+			"libclrjit.so",
+			"libcorclr.so",
+		};
```

Comment:
Can we put this list in a private MSBuild item group? That way if something goes wrong in a release, a customer could add an item as a workaround if needed.

I know we'd lose the ordinal `HashSet`, but the list is short, so might not even matter.

Reply:
Good idea, I can still populate the `HashSet` from it.